auth0-extensions/auth0-account-link-extension
# Fix socket hang up error

The Account Linking Extension was throwing an error if settings where changed after a slightly delayed period. Webtask storage was not being re-read before written. Fix is to force the write, which is the preferred method.

# ✏️ Changes
Set webtask storage context update to `force: true`, based on https://github.com/auth0-extensions/auth0-extension-tools#storage

# 🔗 References
Zendesk: https://auth0.zendesk.com/agent/tickets/42130
CMS: https://auth0team.atlassian.net/browse/CMS-307

# 🎯 Testing
- Open Auth0 Account Link extension
- Add title, click save. This should work fine.
- Wait 1 minute, don't change any fields, click save.
- Error will no longer be thrown.

✅ This has been tested locally
✅ This has been tested in a webtask
✅This change has unit test coverage
✅This change has been acceptance tested
✅This change has been manually smoke tested

# 🚀 Deployment
🚫 This change can support multiple releases of the code serving traffic at the same time
✅ This can be deployed any time because it has no prerequisites.

# 🎡 Rollout

✅ The change will be available to the complete environment at once
🚫 The change is behind a feature flag and will go through a percentage based rollout process
🚫 The change is behind a flag and customers will opt-in to the change

In order to verify that the deployment was successful we will update the extensions via a test tenant and perform smoke tests to ensure proper execution.

# 🔥 Rollback
We will rollback if:

- the smoke test fails
- we receive a customer ticket indicating we broke the extension

## 📄 Procedure

A code revert+redeploy can be done to revert the change.

# 🖥 Appliance

✅ This change is compatible with the appliance.
